---
layout: archive
title: "論文業績一覧"
permalink: /publications/
author_profile: true
---

[2022](#2022), [2021](#2021), [2020](#2020), [2019](#2019), [2018](#2018), [2017](#2017), [2016](#2016), [2015](#2015), [2014](#2014), [2013](#2013), [2012](#2012), [2011](#2011), [2010](#2010), [2009](#2009), [2008](#2008), [2007](#2007), [2006](#2006), [2005](#2005), [2004](#2004), [2003](#2003), [2002](#2002), [2001](#2001), [2000](#2000), [1999](#1999), [1998](#1998), [1997](#1997), [1999](#1996), [1995](#1995) 

# 2022
1.  <u>T. Yoshizawa</u>, <u>S. Ishida</u>, T. Sato, M. Ohta, T. Honma, <u>K. Terayama</u>, ["Selective Inhibitor Design for Kinase Homologs Using Multiobjective Monte Carlo Tree Search,"](https://doi.org/10.1021/acs.jcim.2c00787) _Journal Chemical Information and Modeling_, 2022. 
2.  H. Iwata, Y. Hayashi, A. Hasegawa, <u>K. Terayama</u>, Y. Okuno, ["Classification of scanning electron microscope images of pharmaceutical excipients using deep convolutional neural networks with transfer learning,"](https://doi.org/10.1016/j.ijpx.2022.100135) _International Journal of Pharmaceutics: X_, Vol. 4, 100135, 2022.
3.  T. Tanaka+, <u>T. Ekimoto</u>+, M. Nagatomo, M. Neyazaki, <u>E. Shimoji</u>, T. Yamane, S. Kanagawa, R. Oi, E. Mihara, J. Takagi, S. Akashi, <u>M. Ikeguchi</u>, T. Nogi, ["Hybrid in vitro/in silico analysis of low-affinity protein-protein interactions that regulate signal transduction by Sema6D"](https://onlinelibrary.wiley.com/doi/10.1002/pro.4452), *Protein Science*, in press.　+:  equal contribution
4. M. Sumita, <u>K. Terayama</u>, R. Tamura, K. Tsuda, ["QCforever: A Quantum Chemistry Wrapper for Everyone to Use in Black-Box Optimization,"](https://doi.org/10.1021/acs.jcim.2c00812) _Journal Chemical Information and Modeling_, 62(18), 4427-4434, 2022.
5. <u>K. Osaki</u>, <u>T. Ekimoto</u>, T. Yamane, and <u>M. Ikeguchi</u>, ["3D-RISM-AI: A machine learning approach to predict protein-ligand binding affinity using 3D-RISM"](https://pubs.acs.org/doi/10.1021/acs.jpcb.2c03384), *The Journal of Physical Chemistry B*, 126(33), 6148-6158, 2022
6. Z. Zhang, N. Nomura, Y. Muramoto, <u>T. Ekimoto</u>, T. Uemura, K. Liu, M. Yui, N. Kono, J. Aoki, <u>M. Ikeguchi</u>, T. Noda, S. Iwata, U. Ohto, and T. Shimizu, ["Structure of SARS-CoV-2 membrane protein essential for virus assembly"](https://www.nature.com/articles/s41467-022-32019-3), *Nature Communications* 13, 4399, 2022.
7. S. Nojima<sup>†</sup>, <u>S. Ishida</u>, <u>K. Terayama</u>, K. Matsumoto, T. Matsui, S. Tahara, K. Ohshima, H. Kiyokawa, K. Kido, K. Ukon, S. Y. Yoshida, T. T. Mitani, Y. Doki, T. Mizushima, Y. Okuno, E. A. Susaki, H. R. Ueda, E. Morii<sup>†</sup>, ["A novel three-dimensional imaging system based on polysaccharide staining for accurate histopathological diagnosis of inflammatory bowel diseases,"](https://doi.org/10.1016/j.jcmgh.2022.07.001) *Cellular and Molecular Gastroenterology and Hepatology*, 14(4), 905-924, 2022.
8. A. Tokuhisa<sup>†</sup>, Y. Akinaga, <u>K. Terayama</u>, Y. Okamoto, and Y. Okuno, ["Single-Image Super-Resolution Improvement of X-ray Single-Particle Diffraction Images Using a Convolutional Neural Network,"](https://doi.org/10.1021/acs.jcim.2c00660) *Journal of Chemical Information and Modeling*, 62(14), 3352-3364, 2022.
9. N. Ienaga<sup>†</sup>, K. Higuchi, T. Takashi, K. Gen, <u>K. Terayama</u><sup>†</sup>, ["Normal hatching rate estimation for bulk samples of Pacific bluefin tuna (Thunnus orientalis) eggs using deep learning,"](https://doi.org/10.1016/j.aquaeng.2022.102274) *Aquacultural Engineering*, Vol.98, 102274, 2022.
10. T. Yoshida<sup>†</sup>, J. Zhou, <u>K. Terayama</u>, D. Kitazawa, ["Monitoring of cage-cultured sea cucumbers using an underwater time-lapse camera and deep learning-based image analysis,"](https://doi.org/10.1016/j.atech.2022.100087) *Smart Agricultural Technology*, Vol. 3, 100087, 2022.
11. K. Sato<sup>†</sup>, R. Sasaki, R. Matsuda, <u>M. Nakagawa</u>, <u>T. Ekimoto</u>, T. Yamane, <u>M. Ikeguchi</u>, K. V. Tabata, H. Noji, and K. Kinbara<sup>†</sup>, [“Supramolecular Mechanosensitive Potassium Channel Formed by Fluorinated Amphiphilic Cyclophane,”](https://pubs.acs.org/doi/10.1021/jacs.2c04118) *Journal of the American Chemical Society*, 144(26), 11802-11809, 2022; [Correction](https://pubs.acs.org/doi/10.1021/jacs.2c07339): *Journal of the American Chemical Society*, 2022, DOI: 10.1021/jacs.2c07339
12. Y. Motoyama<sup>†</sup>, R. Tamura<sup>†</sup>, K. Yoshimi<sup>†</sup>, <u>K. Terayama</u>, T. Ueno, K. Tsuda, ["Bayesian optimization package: PHYSBO,"](https://doi.org/10.1016/j.cpc.2022.108405) *Computer Physics Communications*, Vol.278, 108405, 2022.
13. T. Fujita#, <u>K. Terayama</u>#, M. Sumita<sup>†</sup>, R. Tamura, Y. Nakamura, M. Naito, Koji Tsuda<sup>†</sup>, ["Understanding the evolution of a de novo molecule generator via characteristic functional group monitoring,"](https://doi.org/10.1080/14686996.2022.2075240) *Science and Technology of Advanced Materials*, Vol.23, No.1, pp.352-360, 2022.
14.  R. Tamura<sup>†</sup>, G. Deffrennes, K. Han, T. Abe, H. Morito, Y. Nakamura, M. Naito, R. Katsube, Y. Nose, <u>K. Terayama</u>, ["Machine-learning-based phase diagram construction for high-throughput batch experiments,"](https://doi.org/10.1080/27660400.2022.2076548) *Science and Technology of Advanced Materials: Methods*, 2:1, 153-161, 2022.
15. R. Kanada<sup>†</sup>, <u>K. Terayama</u><sup>†</sup>, A. Tokuhisa, S. Matsumoto, Y. Okuno, ["Enhanced Conformational Sampling with an Adaptive Coarse-Grained Elastic Network Model Using Short-Time All-Atom Molecular Dynamics,"](https://pubs.acs.org/doi/10.1021/acs.jctc.1c01074) *Journal of Chemical Theory and Computation*, 18(4), 2062-2074, 2022.
16. M. Sumita, <u>K. Terayama</u>, N. Suzuki, S. Ishihara, R. Tamura, M. K. Chahal, D. T. Payne, K. Yoshizoe, K. Tsuda,  ["De novo creation of a naked eye–detectable fluorescent molecule based on quantum chemical computation and machine learning,"](https://www.science.org/doi/10.1126/sciadv.abj3906) *Science Advances*, Vol.8, No.10, eabj3906, 2022. ([プレスリリース](https://www.yokohama-cu.ac.jp/news/2021/20220310terayama.html))
17. <u>S. Ishida</u>, <u>K. Terayama</u>, R. Kojima, K. Takasu, Y. Okuno, ["AI-Driven Synthetic Route Design Incorporated with Retrosynthesis Knowledge,"](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01074) *Journal of Chemical Information and Modeling*, 62(6), 1357-1367, 2022.
18. G. Deffrennes, <u>K. Terayama</u>, T. Abe, R. Tamura, ["A machine learning–based classification approach for phase diagram prediction,"](https://doi.org/10.1016/j.matdes.2022.110497) *Materials & Design*, Vol.215, 110497, 2022.
19. N. Ienaga, A. Cravotta, A., <u>K. Terayama</u>,  B. W. Scotney, H. Saito, M. G. Busà, ["Semi-automation of gesture annotation by machine learning and human collaboration."](https://link.springer.com/article/10.1007/s10579-022-09586-4) *Language Resources and Evaluation*, 2022.
20. Wei-Hsun Hu, Ta-Te Chen, Ryo Tamura, <u>Kei Terayama</u>, Siqian Wang, Ikumu Watanabe, Masanobu Naito, ["Topological alternation from structurally adaptable to mechanically stable crosslinked polymer,"](https://www.tandfonline.com/doi/full/10.1080/14686996.2021.2025426) *Science and Technology of Advanced Materials*, Vol.23, No.1, pp.66-75, 2022.
21. <u>K. Terayama</u>, K. Mizuno, S. Tabeta, S. Sakamoto, Y. Sugimoto, K. Sugimoto, H. Fukami, M. Sakagami, L. A. Jimenez, ["Cost-effective seafloor habitat mapping using a portable speedy sea scanner and deep-learning-based segmentation: A sea trial at Pujada Bay, Philippines,"](https://doi.org/10.1111/2041-210X.13744) *Methods in Ecology and Evolution*, Vol.13, No.2, pp.339-345, 2022.
22. <u>K. Terayama</u>, K. Han, R. Katsube, I. Ohnuma, T. Abe, Y. Nose, R. Tamura, ["Acceleration of phase diagram construction by machine learning incorporating Gibbs' phase rule,"](https://www.sciencedirect.com/science/article/pii/S1359646221006151) *Scripta Materialia*, Vol. 28, 114335, 2022.


# 2021
1. Xiaolin Sun, Ryo Tamura, Masato Sumita, Kenichi Mori, <u>Kei Terayama</u>, and Koji Tsuda, "Integrating Incompatible Assay Data Sets with Deep Preference Learning," *ACS Medicinal Chemistry Letters*, 13(1): 70-75, (2021). 
1. Y. Amamoto, H. Kikutake, K. Kojio, A. Takahara, and <u>K. Terayama</u>, "Visualization of judgment regions in convolutional neural networks for X-ray diffraction and scattering images of aliphatic polyesters," *Polymer Journal*, 53:1269-1279, (2021).
1. S. Kori#, <u>Y. Shibahashi</u>#, <u>T. Ekimoto</u>#, A. Nishiyama, S. Yoshimi, K. Yamaguchi, S. Nagatoishi, M. Ohta, K. Tsumoto, M. Nakanishi, P. Defossez, <u>M. Ikeguchi</u>, and K. Arita, "Structure-based screening combined with computational and biochemical analyses identified the inhibitor targeting the binding of DNA ligase 1 to UHRF1." Bioorganic Med. Chem. 52, 116500 (2021). # equal contribution
1. <u>K. Moritsugu</u>, "Multiscale Enhanced Sampling Using Machine Learning." Life (Basel). 11(10):1076 (2021).
1. K. Kakihara, K. Asamizu, <u>K. Moritsugu</u>, M. Kubo, T. Kitaguchi, A.Endo, <u>A. Kidera</u>, <u>M. Ikeguchi</u>, A. Kato, M. Komada, T. Fukushima, "Molecular basis of ubiquitin-specific protease 8 autoinhibition by the WW-like domain." Commun Biol. 4(1):1272 (2021).
1. SR. Nasrin, C. Ganser, S. Nishikawa, AMR. Kabir, K. Sada, T. Yamashita, <u>M. Ikeguchi</u>, T. Uchihashi, H. Hess, A. Kakugo, "Deformation of microtubules regulates translocation dynamics of kinesin." Sci Adv. 7(42) (2021).
1. <u>A. Kidera</u>, <u>K. Moritsugu</u>, <u>T.Ekimoto</u>, <u>M.Ikeguchi</u>, "Allosteric regulation of 3CL protease of SARS-CoV-2 and SARS-CoV observed in the crystal structure ensemble." J Mol Biol. 433(24):167324 (2021).
1. <u>K. Terayama</u>, M.Sumita, M.Katouda, K. Tsuda, Y. Okuno, "Efficient Search for Energetically Favorable Molecular Conformations against Metastable States via Gray-Box Optimization." J Chem Theory Comput. 17(8):5419-5427 (2021).
1. B. Ma, <u>K. Terayama</u>, S. Matsumoto, Y. Isaka, Y. Sasakura, H. Iwata, M. Araki, Y. Okuno, "Structure-Based de Novo Molecular Generator Combined with Artificial Intelligence and Docking Simulations." J Chem Inf Model. 61(7):3304-3313 (2021).
1. <u>K. Takedomi</u>, M. Ohta, <u>T. Ekimoto</u>, <u>M. Ikeguchi</u>, "Effect of Water Molecules on the Activating S810L Mutation of the Mineralocorticoid Receptor." J Chem Inf Model. 61(7):3583-3592 (2021).
1. <u>T. Ekimoto</u>, T. Kudo, T. Yamane, <u>M. Ikeguchi</u>, "Mechanism of Vitamin D Receptor Ligand-Binding Domain Regulation Studied by gREST Simulations." J Chem Inf Model. 61(7):3625-3637 (2021).
1. <u>K. Moritsugu</u>, K. Takeuchi, N. Kamiya, J. Higo, I. Yasumatsu, Y. Fukunishi, I. Fukuda, "Flexibility and Cell Permeability of Cyclic Ras-Inhibitor Peptides Revealed by the Coupled Nosé-Hoover Equation." J Chem Inf Model. 61(4):1921-1930 (2021).
1. R. Tamura-Sakaguchi, R. Aruga, M. Hirose, <u>T. Ekimoto</u>, T. Miyake, Y. Hizukuri, R. Oi, M. K. Kaneko, Y. Kato, Y. Akiyama, <u>M. Ikeguchi</u>, K. Iwasaki, T. Nogi, "Moving toward generalizable NZ-1 labeling for 3D structure determination with optimized epitope tag insertion." ACTA Crystallogr. D # equally contribution (2021).
1. <u>K. Moritsugu</u>, N. Yamamoto, Y. Yonezawa, S. I. Tate, H. Fujisaki, "Path Ensembles for Pin1-Catalyzed Cis-Trans Isomerization of a Substrate Calculated by Weighted Ensemble Simulations." J Chem Theory Comput. 17(4):2522-2529 (2021).
1. <u>K. Terayama</u>, M. Sumita, R. Tamura, K. Tsuda, "Black-Box Optimization for Automated Discovery." Acc Chem Res. 54(6):1334-1346 (2021).
1. M. Okuda, <u>T. Ekimoto</u>, J. I. Kurita, <u>M. Ikeguchi</u>, Y. Nishimura, "Structural and dynamical insights into the PH domain of p62 in human TFIIH." Nucleic Acids Res. 49(5):2916-2930 (2021).
1. S. Matsumoto, <u>S. Ishida</u>, M. Araki, T. Kato, <u>K. Terayama</u>, Y. Okuno, "Extraction of protein dynamics information from cryo-EM maps using deep learning." Nature Machine Intelligence. 3(2):153-160 (2021).
1. M. Mori, K. Sato, <u>T. Ekimoto</u>, A. Okumura, <u>M. Ikeguchi</u>, K. V. Tabata, H. Noji, K. Kinbara, "Imidazolinium-based Multiblock Amphiphile as Transmembrane Anion Transporter." Chem Asian J. 16(2):147-157 (2021).
1. H. Hagihara, N. Ienaga, <u>K. Terayama</u>, Y. Moriguchi, M. A. Sakagami, "Looking represents choosing in toddlers: Exploring the equivalence between multimodal measures in forced-choice tasks." Infancy. 26(1):148-167 (2021).
1. N. Ienaga, K. Higuchi, T. Takashi, K. Gen, K. Tsuda, <u>K. Terayama</u>, "Vision-based egg quality prediction in Pacific bluefin tuna (Thunnus orientalis) by deep neural network." Sci Rep. 11(1):6 (2021).
1. I. Fukuda, <u>K. Moritsugu</u> and Y. Fukunishi, "On ergodicity for multi-dimensional harmonic oscillator systems with Nose-Hoover type thermostat", Regular and Chaotic Dynamics. 26: 183-204 (2021).


# 2020
1. R. Shibukawa, S. Ishida, K. Yoshizoe, K. Wasa, K. Takasu, Y. Okuno, K. Terayama, and K. Tsuda, "CompRet: a comprehensive recommendation framework for chemical synthesis planning with algorithmic enumeration." Journal of Cheminformatics 12, 52, (2020).
1. J. Zhang, K. Terayama, M. Sumita, K. Yoshizoe, K. Ito, J. Kikuchi, and K. Tsuda, "NMR-TS: de novo molecule identification from NMR spectra." STAM, 21, 1, 552-561 (2020).
1. A. Tokuhisa, R. Kanada, S. Chiba, K. Terayama, Y. Isaka, B. Ma, N. Kamiya, and Y. Okuno, "Coarse-grained diffraction template matching model to retrieve multi-conformational models for biomolecule structures from noisy diffraction patterns." J. Chem. Inf. Model. 60, 6, 2803-2818 (2020).
1. S. Tojo, Z. Zhang, H. Matsui, M. Tahara, M. Ikeguchi, M. Kochi, M. Kamada, H. Shigematsu, A. Tsutsumi, N. Adachi, T. Shibata, M. Yamamoto, M. Kikkawa, T. Senda, Y. Isobe, U. Ohto, T. Shimizu, "Structural analysis reveals TLR7 dynamics underlying antagonism." Nat. Commun. 11(1):5204 (2020).
1. Yoshidome T, Ikeguchi M, Ohta M, "Comprehensive 3D‐RISM analysis of the hydration of small molecule binding sites in ligand‐free protein structures." J Comput. Chem. 41(28):2406-2419 (2020).
1. K. Ito, Y. Murayama, Y. Kurokawa, S. Kanamaru, Y. Kokabu, T. Maki, T. Mikawa, B. Argunhan, H. Tsubouchi, M. Ikeguchi, M. Takahashi, H. Iwasaki, " Real-time tracking reveals catalytic roles for the two DNA binding sites of Rad51." Nat. Commun. 11(1):2950 (2020).
1. T. Muraoka, D. Noguchi, R. S. Kasai, K. Sato, R. Sasaki, K. V. Tabata, T. Ekimoto, M. Ikeguchi, K. Kamagata, N. Hoshino, H. Noji, T. Akutagawa, K. Ichimura, K. Kinbara, "A synthetic ion channel with anisotropic ligand response." Nat. Commun. 11(1):2924 (2020).
1. K. Kato, T. Masuda, C. Watanabe, N. Miyagawa, H. Mizouchi, S. Nagase, K. Kamisaka, K. Oshima, S. Ono, H. Ueda, A. Tokuhisa, R. Kanada, M. Ohta, M. Ikeguchi, Y. Okuno, K. Fukuzawa, T. Honma, "High-Precision Atomic Charge Prediction for Protein Systems Using Fragment Molecular Orbital Calculation and Machine Learning." J. Chem. Inf. Model. 60, 7, 3361-3368 (2020).
1. S. Kori, T. Jimenji, T. Ekimoto, M. Sato, F. Kusano, T. Oda, M. Unoki, M. Ikeguchi, K. Arita, "Serine 298 Phosphorylation in Linker 2 of UHRF1 Regulates Ligand-Binding Property of Its Tans Tudor Domain." J. Mol. Biol, 432(14):4061-4075 (2020).
1. M. Inoue, T. Hayashi, S. Hikiri, M. Ikeguchi, M. Kinoshita, "Hydration properties of a protein at low and high pressures: Physics of pressure denaturation." J Chem Phys., 152(6):065103 (2020).
1. K. Terayama, M. Sumita, R. Tamura, D. T. Payne, M. K. Chahal, S. Ishihara, K. Tsuda, "Pushing property limits in materials discovery via boundless objective-free exploration." Chem Sci., 11(23):5959-5968 (2020).
1. K. Mizuno, K. Terayama, S. Hagino, S. Tabeta, S. Sakamoto, T. Ogawa, K. Sugimoto, H. Fukami, "An efficient coral survey method based on a large-scale 3-D structure model obtained by Speedy Sea Scanner and U-Net segmentation." Sci Rep 31;10(1):12416 (2020).
1. K. Moritsugu, Y. Nishino, A. Kidera, "Inter-lobe Motions Allosterically Regulate the Structure and Function of EGFR Kinase." J Mol Biol. 24;432(16):4561-4575 (2020).
1. I. Fukuda and K. Moritsugu, "Temperature-Energy-space Sampling Molecular Dynamics: Deterministic, Iteration-free, and Single-replica Method utilizing Continuous Temperature System." J. Phys. 53: 375004 (2020).
1. R. Katsube, K. Terayama, R. Tamura, and Y. Nose, "Experimental establishment of phase diagram guided by uncertainty sampling: an application to the deposition of Zn-Sn-P films by molecular beam epitaxy." ACS Materials Letters, Vol. 2, 571-575 (2020).
1. H. Hagihara, N. Ienaga, D. Enomoto, S. Takahata, H. Ishihara, H. Noda, K. Tsuda, and K. Terayama, "Computer Vision-Based Approach for Quantifying Occupational Therapists' Qualitative Evaluations of Postural Control." Occup Ther Int, vol.2020, Article ID 8542191, 9 pages (2020).
1. R. Kanada, A. Tokuhisa, K. Tsuda, Y. Okuno, K. Terayama, "Exploring Successful Parameter Region for Coarse-Grained Simulation of Biomolecules by Bayesian Optimization and Active Learning." Biomolecules, 10(3), 482, (2020).
1. 浴本亨, 小甲裕一, 苙口友隆, 池口満徳 "X線小角散乱と分子動力学シミュレーションを組み合わせた生体分子の溶液構造解析" 分子シミュレーション学会誌"アンサンブル", 22, 3, 239-251 (2020).

# 2019
1. K. Moritsugu, T. Ito and A. Kidera, "Allosteric response to ligand binding: Molecular dynamics study of the N-terminal domains in IP3 receptor." Biophys Physicobiol. 16, 232-239 (2019).
1. A. Singharoy, C. Chipot, T. Ekimoto, K. Suzuki, M. Ikeguchi, I. Yamato, and T. Murata, "Rotational mechanism model of the bacterial V1 motor based on structural and computational analyses." Front. Physiol. 10:46, 1-12 (2019).
1. S. Hikiri, T. Hayashi, M.Inoue, T. Ekimoto, M. Ikeguchi, and M. Kinoshita, "An accurate and rapid method for calculating hydration free energies of a variety of solutes including proteins." J. Chem. Phys. 150, 175101-1-12 (2019).
1. T. Ekimoto, Y. Kokabu, T. Oroguchi, M. Ikeguchi, "Combination of coarse-grained molecular dynamics simulations and small-angle x-ray scattering experiments." Biophys. Physicobiol. 16, 377-390 (2019).
1. S Chiba, Y Okuno, T Honma, M Ikeguchi, "Force-field parametrization based on radial and energy distribution functions." J Comput. Chem., 5, 40, 29, 2577-2585 (2019).
1.　T. Yamada, T. Hayashi, S. Hikiri, N. Kobayashi, H. Yanagawa, M. Ikeguchi, M. Katahira, T. Nagata, M. Kinoshita, "How Does the Recently Discovered Peptide MIP Exhibit Much Higher Binding Affinity than an Anticancer Protein p53 for an Oncoprotein MDM2?" J. Chem. Inf. Model., 59, 8, 3533-3544 (2019).

# 2018
1. Y. Matsunaga, T. Yamane, T. Terada, K. Moritsugu, H. Fujisaki, S. Murakami, M. Ikeguchi, and A. Kidera, "Energetics and conformational pathways of functional rotation in the multidrug transporter AcrB." eLIFE (2018) 7: e31715. DOI: 10.7554/eLife.31715.001 (2018).
1. K. Moritsugu, T. Terada, H. Kokubo, S. Endo, T. Tanaka and A. Kidera, "Multiscale enhanced sampling of glucokinase: Regulation of the enzymatic reaction via a large scale domain motion." Journal of Chemical Physics 149, 072314. DOI: 10.1063/1.5027444 (2018).
1. H. Fujisaki, K. Moritsugu, A. Mitsutake, and H. Suetani, "Conformational change of a biomolecule studied by the weighted ensemble method: Use of the diffusion map method to extract reaction coordinates." Journal of Chemical Physics 149, 134112. DOI: 10.1063/1.5049420 (2018).
1. K. Moritsugu, H. Nishi, K. Inariyama, M. Kobayashi and A. Kidera, "Dynamic recognition and linkage specificity in K63 di-ubiquitin and TAB2 NZF domain complex." Scientific Reports, 8, 16478. DOI: 10.1038/s41598-018-34605-2 (2018).
1. H. Fujisaki, K. Moritsugu and Y. Matsunaga, "Exploring configuration space and path space of biomolecules using enhanced sampling techniques: Searching for mechanism and kinetics of biomolecular functions." International Journal of Molecular Sciences 19, 3177. DOI: 10.3390/ijms19103177(2018).
1. T. Ekimoto and M. Ikeguchi, "Multiscale Molecular Dynamics Simulations of Rotary Motor Proteins." Biophys. Rev. 10, 605-615 (2018).
1. T. Ekimoto, T.Yamane, and M. Ikeguchi, "Elimination of Finite-Size Effects on Binding Free Energies via the Warp-Drive Method." J. Chem. Theory Comput., 14, 6544-6559 (2018).
1. T. Ekimoto and M. Ikeguchi, "Hybrid Methods for Modeling Protein Structures Using Molecular Dynamics Simulations and Small-Angle X-Ray Scattering Data." Chapter 15, pp237-258, Integrative Structural Biology with Hybrid Methods, Springer, 2018, H.Nakamura, G.Kleywegt, S.Burley, J.L.Markley (Eds.)
1. Yonekura K, Matsuoka R, Yamashita Y, Yamane T, Ikeguchi M, Kidera A, Maki-Yonekura S., "Ionic scattering factors of atoms that compose biological molecules." IUCrJ 5 (Pt 3) 348-353 (2018).
1. Lee JG, Youn HS, Kang JY, Park SY, Kidera A, Yoo YJ, Eom SH. , "Crystal structure of the Ube2K/E2-25K and K48-linked di-ubiquitin complex provides structural insight into the mechanism of K48-specific ubiquitin chain synthesis." Biochem Biophys Res Commun. , 17;506(1):102-107. (2018).
1. S Hikiri, T Hayashi , M Ikeguchi , M Kinoshita. , "Statistical thermodynamics for the unexpectedly large difference between disaccharide stereoisomers in terms of solubility in water." Phys Chem Chem Phys , 20(36):23684-23693 , (2018).
1. Isobe Y, Itagaki M, Ito Y, Naoe S, Kojima K, Ikeguchi M, Arita M. "Comprehensive analysis of the mouse cytochrome P450 family responsible for omega-3 epoxidation of eicosapentaenoic acid." Sci Rep , 8(1):7954. , (2018).
1. Kajiwara Y, Yasuda S, Hikiri S, Hayashi T, Ikeguchi M, Murata T, Kinoshita M. , "Physical Origin of Thermostabilization by a Quadruple Mutation for the Adenosine A2a Receptor in the Active State." J Phys. Chem B. , 122(16):4418-4427., (2018).

# 2017
1. Y. Isaka, T. Ekimoto, Y. Kokabu, I. Yamato, T. Murata, M. Ikeguchi, "Rotation Mechanism of Molecular Motor V1-ATPase Studied by Multiscale Molecular Dynamics Simulation", Biophys. J., 112, 911-920 (2017).
1. K. Moritsugu, T. Terada, A. Kidera, "Free-Energy Landscape of Protein-Ligand Interactions Coupled with Protein Structural Changes,", J. Phys. Chem. B, 121, 731-740 (2017).
1. Satoshi Ishiyama, Atsuya Nishiyama, Yasushi Saeki, Kei Moritsugu, Daichi Morimoto, Luna Yamaguchi, Naoko Arai, Rumie Matsumura, Toru Kawakami, Yuichi Mishima, Hironobu Hojo, Shintaro Shimamura, Fuyuki Ishikawa, Shoji Tajima, Keiji Tanaka, Mariko Ariyoshi, Masahiro Shirakawa, Mitsunori Ikeguchi, Akinori Kidera, Isao Suetake, Kyohei Arita, and Makoto Nakanishi, "Structure of the H3 ubiquitin code/Dnmt1 reader module complex reveals the basis for DNA methylation maintenance." Molecular Cell, (2017) 68: 350-360. DOI: 10.1016/j.molcel.2017.09.037

# 2016
1. S. Hikiri, T. Yoshidome, M. Ikeguchi, "Computational Methods for Configurational Entropy using Internal and Cartesian Coordinates", J. Chem. Theory Compt., 12, 5990-6000 (2016).
1. K. Moritsugu, T. Terada, A. Kidera, "Multiscale enhanced sampling for protein systems: An extension via adiabatic separation", Chem. Phys. Lett., 661, 279-283 (2016).
1. M. Ota, M. Ikeguchi, A. Kidera, "Itinerary profiling to analyze a large number of protein-folding trajectories", Biophysics and Physicobiology, 13, 295-304 (2016).
1. I. Fukuda, K. Moritsugu, "Coupled Nosé-Hoover equations of motions without time scaling", J. Phys. A, 50, 015002 (2016).
1. Y. Anami, N. Shimizu, T. Ekimoto, D. Egawa, T. Itoh, M. Ikeguchi, K. Yamamoto, "Apo- and antagonist-binding structures of vitamin D receptor ligand-binding domain revealed by hybrid approach combining small-angle X-ray scattering and molecular dynamics", J. Med. Chem., 59, 7888-7900 (2016).
1. H. Shimojo, A. Kawaguchi, T. Oda, N. Hashiguchi, S. Omori, K. Moritsugu, A. Kidera, K. Hiragami-Hamada, J. Nakayama, M. Sato, Y. Nishimura, "Extended string-like binding of the phosphorylated HP1α N-terminal tail to the lysine 9-methylated histone H3 tail", Sci. Rep., 6, 22527 (2016).
1. I. Fukuda, K. Moritsugu, "Coupled Nosé-Hoover Equations of Motions: coupling the physical system Nosé-Hoover equation and temperature system Nosé-Hoover equation", Phys. Rev. E, 93, 033306 (2016).

# 2015
1. T. Yoshidome, T. Oroguchi, M. Nakasako, M. Ikeguchi, "Classification of projection images of proteins with structural polymorphism by manifold: A simulation study for X-Ray Free-Electron Laser Diffraction Imaging", Phys. Rev. E, 92, 032710 (2015).
1. T. Yoshidome, T. Ekimoto, N. Matubayasi, Y. Harano, M. Kinoshita, M. Ikeguchi, "An Accurate and Efficient Computation Method of the Hydration Free Energy of a Large, Complex Molecule", J. Chem. Phys., 142, 175101 (2015).
1. T. Ekimoto, N. Matubayasi, M. Ikeguchi, "Finite-size effect on the charging free energy of protein in explicit solvent", J.Chem.Theory Comput., 11, 215-223 (2015).
1. Y. Ito, M. Ikeguchi, "Mechanism of the αβ conformational change in F1-ATPase after ATP hydrolysis: free-energy simulations", Biophys. J., 108, 85-97 (2015).
1. K. Saikusa, A. Nagadoi, K. Hara, S. Fuchigami, H. Kurumizaka, Y. Nishimura, S. Akashi, "Mass spectrometric approach for characterizing the disordered tail regions of the histone H2A/H2B dimer", Anal. Chem., 87, 2220-2227 (2015).
1. K. Moritsugu, R. Koike, K. Yamada, H. Kato, A. Kidera, "Motion Tree delineates hierarchical structure of protein dynamics observed in molecular dynamics simulation", PLoS ONE, 10, e0131583 (2015).
1. H. Fujisaki, K. Moritsugu, Y. Matsunaga, T. Morishita, L. Maragliano, "Extended phase-space methods for enhanced sampling in molecular simulations: a review", Frontiers in Bioengineering and Biotechnology, 3, 125 (2015).
1. I. Fukuda, K. Moritsugu, "Double Density Dynamics: Dynamical Realization of Superstatistics", J. Phys. A, 48, 455001 (2015).

# 2014
1. R. Koike, M. Ota, A. Kidera, "Hierarchical description and extensive classification of protein structural changes by motion tree", J. Mol. Biol., 426, 752-762 (2014).
1. K. Moritsugu, A. Kidera, J. C. Smith, "Solvent friction effects propagate over the entire protein molecule through low-frequency collective modes", J. Phys. Chem. B, 118, 8559-8565 (2014).
1. K. Moritsugu, T. Terada, A. Kidera, "Energy landscape of all-atom protein-protein interactions revealed by multiscale enhanced sampling", PLoS Comput. Biol., 10, e1003901 (2014).
1. K. Moritsugu, T. Terada, A. Kidera, "Multiscale enhanced sampling driven by multiple coarse-grained models", Chem. Phys. Lett., 616, 20-24 (2014).
1. 森次 圭 "マルチスケール手法によるタンパク質全原子構造サンプリング" 分子シミュレーション研究会会誌"アンサンブル" 16, 227-233 (2014).

# 2013
1. K. Saikusa, N. Kuwabara, Y. Kokabu, Y. Inoue, M. Sato, H. Iwasaki, T. Shimizu, M. Ikeguchi, S. Akashi, "Characterisation of an intrinsically disordered protein complex of Swi5-Sfr1 by ion mobility mass spectrometry and small-angle X-ray scattering", Analyst, 138, 1441-1449 (2013).
1. Y. Ito, T. Yoshidome, N. Matubayasi, M. Kinoshita, M. Ikeguchi, "Molecular dynamics simulations of yeast F1-ATPase before and after 16° rotation of the γ subunit", J. Phys. Chem. B, 117, 3298-3307 (2013).
1. Y. Kokabu, M. Ikeguchi, "Molecular modeling and molecular dynamics simulations of recombinase Rad51", Biophys. J., 104, 1556-1565 (2013).
1. K. Saikusa, S. Fuchigami, K. Takahashi, Y. Asano, A. Nagadoi, H. Tachiwana, H. Kurumizaka, M. Ikeguchi, Y. Nishimura, S. Akashi, "Gas-phase structure of the histone multimers characterized by ion mobility mass spectrometry and molecular dynamics simulation", Anal. Chem., 85, 4165-4171 (2013).
1. H. Fujisaki, M. Shiga, K. Moritsugu, A. Kidera, "Multiscale enhanced path sampling based on the Onsager-Machlup action: Application to a model polymer", J. Chem. Phys., 139, 054117 (2013).
1. Y. Ida, A. Kidera, "The conserved Arg241-Glu439 salt bridge determines flexibility of the inositol 1,4,5-trisphosphate receptor binding core in the ligand-free state", Proteins, 81, 1699-1708 (2013).
1. T. Yamane, S. Murakami, M. Ikeguchi, "Functional rotation induced by alternating protonation states in the multidrug transporter AcrB: All-atom molecular dynamics simulations", Biochemistry, 52, 7648-7658 (2013).
1. Y. Naritomi, S. Fuchigami, "Slow dynamics of a protein backbone in molecular dynamics simulation revealed by time-structure based independent component analysis", J. Chem. Phys., 139, 215102 (2013).

# 2012
1. T. Amemiya, R. Koike, A. Kidera, M. Ota, "PSCDB: a database for protein structural change upon ligand binding", Nucleic Acids Res., 40, D554-558 (2012).
1. T. Terada, A. Kidera, "Comparative molecular dynamics simulation study of crystal environment effect on protein structure", J. Phys. Chem. B, 116, 6810-6818 (2012).
1. N. Kuwabara, Y. Murayama, H. Hashimoto, Y. Kokabu, M. Ikeguchi, M. Sato, K. Mayanagi, Y. Tsutsui, H. Iwasaki, T. Shimizu, "Mechanistic insights into the activation of Rad51-mediated strand exchange from the structure of a recombination activator, the Swi5-Sfr1 complex", Structure, 20, 440-449 (2012).
1. K. Moritsugu, T. Terada, A. Kidera, "Disorder-to-order transition of an intrinsically disordered region of sortase revealed by multiscale enhanced sampling", J. Am. Chem. Soc., 134, 7094-7101 (2012).
1. S. Hayashi, H. Ueno, A. R. Shaikh, M. Umemura, M. Kamiya, Y. Ito, M. Ikeguchi, Y. Komoriya, R. Iino, H. Noji, "Molecular mechanism of ATP hydrolysis in F1-ATPase revealed by molecular simulations and single-molecule observations", J. Am. Chem. Soc., 134, 8447-8454 (2012).
1. Y. Matsunaga, H. Fujisaki, T. Terada, T. Furuta, K. Moritsugu, A. Kidera, "Minimum free energy path of ligand-induced transition in adenylate kinase", PLoS Comput. Biol., 8, e1002555 (2012).
1. H. Mishima, S. Yasuda, T. Yoshidome, H. Oshima, Y. Harano, M. Ikeguchi, M. Kinoshita, "Characterization of experimentally determined native-structure models of a protein using energetic and entropic components of free-energy function", J. Phys. Chem. B, 116, 7776-7786 (2012).
1. K. Nishikata, M. Ikeguchi, A. Kidera, "Comparative simulations of the ground state and the M-intermediate state of the sensory rhodopsin II-transducer complex with a HAMP domain model", Biochemistry, 51, 5958-5966 (2012).
1. T. Yoshidome, Y. Ito, N. Matubayasi, M. Ikeguchi, M. Kinoshita, "Structural characteristics of yeast F1-ATPase before and after 16-degree rotation of the γ subunit: theoretical analysis focused on the water-entropy effect", J. Chem. Phys., 137, 035102 (2012).
1. M. Tanigawara, K. V. Tabata, Y. Ito, J. Ito, R. Watanabe, H. Ueno, M. Ikeguchi, H. Noji, "Role of the DELSEED loop in torque transmission of F1-ATPase", Biophys. J., 103, 970-978 (2012).
1. Y. Matsunaga, R. Koike, M. Ota, J. R. Tame, A. Kidera, "Influence of structural symmetry on protein dynamics", PLoS One, 7, e50011 (2012).
1. 森次 圭 "粗視化モデルを用いたマルチスケールシミュレーションシステムの構築" 生物物理学会誌 52, 26-27 (2012).

# 2011
1. C. Motono, J. Nakata, R. Koike, K. Shimizu, M. Shirota, T. Amemiya, K. Tomii, N. Nagano, N. Sakaya, K. Misoo, M. Sato, A. Kidera, H. Hiroaki, T. Shirai, K. Kinoshita, T. Noguchi, M. Ota, "SAHG, a comprehensive database of predicted structures of all human proteins.", Nucleic Acids Res., 39, D487-493 (2011).
1. T. Oroguchi, M. Ikeguchi, "Effects of ionic strength on SAXS data for proteins revealed by molecular dynamics simulations.", J. Chem. Phys., 134, 025102 (2011).
1. Y. Naritomi, S. Fuchigami, "Slow dynamics in protein fluctuations revealed by time-structure based independent component analysis: the case of domain motions.", J. Chem. Phys., 134, 065101 (2011).
1. Y. Ito, T. Oroguchi, M. Ikeguchi, "Mechanism of the Conformational Change of the F1-ATPase β Subunit Revealed by Free Energy Simulations.", J. Am. Chem. Soc., 133, 3372-3380 (2011).
1. T. Yoshidome, Y. Ito, M. Ikeguchi, M. Kinoshita, "Rotation Mechanism of F1-ATPase: Crucial Importance of the Water Entropy Effect.", J. Am. Chem. Soc., 133, 4030-4039 (2011).
1. T. Amemiya, R. Koike, S. Fuchigami, M. Ikeguchi, A. Kidera, "Classification and annotation of the relationship between protein structural change and ligand binding.", J. Mol. Biol., 408, 568-584 (2011).
1. T. Imai, N. Miyashita, Y. Sugita, A. Kovalenko, F. Hirata, A. Kidera, "Functionality mapping on internal surfaces of multidrug transporter AcrB based on molecular theory of solvation: implications for drug efflux pathway.", J. Phys. Chem. B, 115, 8288-8295 (2011).
1. S. Yasuda, T. Yoshidome, Y. Harano, R. Roth, H. Oshima, K. Oda, Y. Sugita, M. Ikeguchi, M. Kinoshita, "Free-energy function for discriminating the native fold of a protein from misfolded decoys.", Proteins, 79, 2161-2171 (2011).
1. H. Oshima, S. Yasuda, T. Yoshidome, M. Ikeguchi, M. Kinoshita, "Crucial importance of the water-entropy effect in predicting hot spots in protein-protein complexes.", Phys. Chem. Chem. Phys., 13, 16236-16246 (2011).
1. K. M. Downard, Y. Kokabu, M. Ikeguchi, S. Akashi, "Homology-modelled structure of the βB2B3-crystallin heterodimer studied by ion mobility and radical probe MS.", FEBS J., 278, 4044-4054 (2011).
1. Y. Kokabu, Y. Murayama, N. Kuwabara, T. Oroguchi, H. Hashimoto, Y. Tsutsui, N. Nozaki, S. Akashi, S. Unzai, T. Shimizu, H. Iwasaki, M. Sato, M. Ikeguchi, "Fission yeast Swi5-Sfr1 protein complex, an activator of Rad51 recombinase, forms an extremely elongated dogleg-shaped structure.", J. Biol. Chem., 286, 43569-43576 (2011).
1. M. Yamamoto, K. Nakagawa, K. Fujiwara, A. Shimizu, M. Ikeguchi, M. Ikeguchi, "A native disulfide stabilizes non-native helical structures in partially folded states of equine β-lactoglobulin.", Biochemistry, 50, 10590-10597 (2011).

# 2010
1. S. Omori, S. Fuchigami, M. Ikeguchi, A. Kidera, "Latent dynamics of a protein molecule observed in dihedral angle space." J. Chem. Phys., 132, 115103 (2010).
H. Fujisaki, M. Shiga, A. Kidera, "Onsager-Machlup action-based path sampling and its combination with replica exchange for diffusive and multiple pathways." J. Chem. Phys., 132, 134101 (2010).
1. I. Yanagihara, K. Nakahira, T. Yamane, S. Kaieda, K. Mayanagi, D. Hamada, T. Fukui, K. Ohnishi, S. Kajiyama, T. Shimizu, M. Sato, T. Ikegami, M. Ikeguchi, T. Honda, H. Hashimoto, "Structure and functional characterization of Vibrio parahaemolyticus thermostable direct hemolysin." J. Biol. Chem., 285, 16267-16274 (2010).
1. D. Okuno, M. Ikeguchi, H. Noji, "Measurement of the conformational state of F1-ATPase by single-molecule rotation." Methods Enzymol., 475, 279-296 (2010).
1. T. Yamane, H. Okamura, Y. Nishimura, A. Kidera, M. Ikeguchi, "Side-chain conformational changes of transcription factor PhoB upon DNA binding: a population-shift mechanism." J. Am. Chem. Soc., 132, 12653-12659 (2010).
1. N. Kuwabara, H. Hashimoto, N. Yamada, S. Unzai, M. Ikeguchi, M. Sato, Y. Murayama, H. Iwasaki, T. Shimizu, "Expression, purification and crystallization of Swi5 and the Swi5-Sfr1 complex from fission yeast." Acta Crystallogr., 66, 1124-1126 (2010).
1. Y. Sugita, M. Ikeguchi, C. Toyoshima, "Relationship between Ca2+-affinity and shielding of bulk water in the Ca2+-pump from molecular dynamics simulations" Proc. Natl. Acad. Sci., 107, 21465-21469 (2010).
1. K. Moritsugu, T. Terada, A. Kidera, "Scalable free energy calculation of proteins via multiscale essential sampling" J. Chem. Phys., 133, 224105 (2010).
1. S. Fuchigami, S. Omori, M. Ikeguchi, A. Kidera, "Normal mode analysis of protein dynamics in a non-eckart frame." J. Chem. Phys., 132, 104109 (2010).
1. K. Nishikata, S. Fuchigami, M. Ikeguchi, A. Kidera, "Molecular modeling of the HAMP domain of sensory rhodopsin II transducer from Natronomonas pharaonis." BIOPHYSICS, 6, 27-36 (2010).
1. S. Omori, S. Fuchigami, M. Ikeguchi, A. Kidera, Latent dynamics of a protein molecule observed in dihedral angle space. J. Chem. Phys., 132, 115103 (2010)
1. Y. Ito, M. Ikeguchi, "Molecular dynamics simulations of the isolated β subunit of F1-ATPase." Chem. Phys. Lett., 490, 80-83 (2010).
1. Y. Ito, M. Ikeguchi, "Structural fluctuation and concerted motions in F1-ATPase: a molecular dynamics study." J. Comput. Chem., in press.
1. I. Yanagihara, K. Nakahira, T. Yamane, S. Kaieda, K. Mayanagi, D. Hamada, T. Fukui, K. Ohnishi, S.i. Kajiyama, T. Shimizu, M. Sato, T. Ikegami, M. Ikeguchi, T. Honda, H. Hashimoto, "Structure and Functional Characterization of Vibrio parahaemolyticus Thermostable Direct Hemolysin (TDH). J. Biol. Chem., in press.
1. H. Yagi, H. Konno, T. Murakami-Fuse, A. Isu, T. Oroguchi, H. Akutsu, M. Ikeguchi, T. Hisabori, "Structural and functional analysis of the intrinsic inhibitor subunit ε of F1-ATPase from photosynthetic organisms." Biochem. J., 425, 85-94 (2010).

# 2009
1. S. Omori, S. Fuchigami, M. Ikeguchi, A. Kidera, "Linear response theory in dihedral angle space for protein structural change upon ligand binding." J. Comput. Chem., 30, 2602-2608 (2009).
1. T. Oroguchi, H. Hashimoto, T. Shimizu, M. Sato, and M. Ikeguchi, "Intrinsic dynamics of restriction endonuclease EcoO109I studied by molecular dynamics simulations and X-Ray scattering data analysis." Biophys. J., 96, 2808-2822 (2009).
1. T. Yoshidome, K. Oda, Y. Harano, R. Roth, Y. Sugita, M. Ikeguchi, M. Kinoshita, "Free-energy function based on an all-atom model for proteins." Proteins: Structure, Function, and Bioinformatics, 77, 950-961 (2009).
1. M. Ikeguchi, Water transport in aquaporins: molecular dynamics simulations. Frontiers in Bioscience, 14, 1283-1291 (2009).

# 2008
1. R. Koike, T. Amemiya, M. Ota, and A. Kidera, "Protein structural change upon ligand binding correlates with enzymatic reaction mechanism." J. Mol. Biol. 379, 397-401 (2008).
1. T. Yamane, H. Okamura, M. Ikeguchi, Y. Nishimura, A. Kidera, "Water-mediated interactions between DNA and PhoB DNA-binding/transactivation domain: NMR-restrained molecular dynamics in explicit water environment." Proteins: Structure, Function, and Bioinformatics, 71, 1970-1983 (2008).
1. 池口満徳, "水を輸送する分子マシン アクアポリンの分子シミュレーション", "最新分子マシン ナノで働く"高度な機械"を目指して", 化学同人編集部, 化学同人, 132-137 (2008).
1. 木寺詔紀, 池口満徳, アクアポリンの水透過過程の比較分子シミュレーション, 生化学, 80, 940-947 (2008).

# 2007
1. T. Oroguchi, M. Ikeguchi, M. Ota, K. Kuwajima, A. Kidera, "Unfolding pathways of goat α-lactalbumin as revealed in multiple alignment of molecular dynamics trajectories", J. Mol. Biol., 371, 1354-1364 (2007).
1. M. Hashido, A. Kidera, M. Ikeguchi, "Water transport in aquaporins: osmotic permeability matrix analysis of molecular dynamics simulations", Biophys. J., 93, 373-385 (2007).
1. Y. Harano, R. Roth, Y. Sugita, M. Ikeguchi, M. Kinoshita, "Physical basis for characterizing native structures of proteins", Chem. Phys. Lett., 437, 112-116 (2007).
1. Y. Akamatsu, Y. Tsutsui, T. Morishita, MD S. P Siddique, Y. Kurokawa, M. Ikeguchi, F. Yamao, B. Arcangioli, H. Iwasaki, "Fission yeast Swi5/Sfr1 and Rhp55/Rhp57 differentially regulate Rhp51-dependent recombination outcomes", EMBO J., 26, 1352-1362 (2007).
1. C. Addy, M. Ohara, F. Kawai, A. Kidera, M. Ikeguchi, S. Fuchigami, M. Osawa, I. Shimada, S.-Y. Park, J. Tame, and J. Heddle, "Nickel binding to NikA: An additional binding site reconciles spectroscopy, calorimetry and crystallography", Acta Cryst. D63, 221-229 (2007)
1. R. Koike, K. Kinoshita, and A. Kidera, "Probabilistic alignment detects remote homology in a pair of protein sequences without using additional information of homologous sequences.", Proteins: Structure, Function, and Bioinformatics, 66, 655-663 (2007)
1. 池口満徳, "生物系の分子動力学シミュレーション", 日本物理学会誌, 62, 777-784 (2007).

# 2006
1. Y. Sugita, N. Miyashita, T. Yoda, M. Ikeguchi, C. Toyoshima, "Structural changes of the cytoplasmic domain of phospholamban by phosphorylation at Ser16: a molecular dynamics study", Biochemistry, 45, 11752-11761 (2006).
1. S. Ki, F. Sugihara, K. Kasahara, H. Tochio, A. Okada-Marubayashi, S. Tomita, M. Morita, M. Ikeguchi, M. Shirakawa, T. Kokubo, "A novel magnetic resonance-based method to measure gene expression in living cells", Nucl. Acids Res., 34, e51 (2006).
1. 池口満徳, 木寺詔紀, "タンパク質構造変化の理論：平衡ゆらぎと線形応答理論", 生物物理, 46, 275-278 (2006).
1. 杉田有治, 池口満徳, "脂質二重膜・膜蛋白質の分子動力学計算", 物性研究, 86, 157-197 (2006).
1. 池口満徳, 木寺詔紀, "蛋白質構造変化の源はゆらぎ-分子結合に伴う蛋白質立体構造変化の理論-", 蛋白質核酸酵素, 51, 268-273 (2006).
1. 太田元規, 池口満徳, "タンパク質フォールディング博物学の曙", 生物物理, 46, 224-227 (2006).

# 2005
1. T. Oroguchi, M. Ikeguchi, K. Saeki, K. Kamagata, Y. Sawano, M. Tanokura, A. Kidera, and K. Kuwajima, "Atomically detailed description of the unfolding of a-lactalbumin by the combined use of experiments and simulations", J. Mol. Biol. 354, 164 172 (2005).
1. M. Hashido, M. Ikeguchi, and A. Kidera, "Comparative Simulations of Aquaporin Family: AQP1, AQPZ, AQP0 and GlpF", FEBS Lett. 579, 5549 5552 (2005).
1. M. Ikeguchi, J. Ueno, M. Sato and A. Kidera, "Protein structural change upon ligand binding: Linear response theory.", Phys. Rev. Lett. 94, 078102 1-4 (2005).
1. Y. Sugita, N. Miyashita, M. Ikeguchi, A. Kidera and C. Toyoshima, "Protonation of the acidic residues in the transmembrane cation-binding sites of the Ca2+-pump.", J. Am Chem. Soc. 127, 6150-6151 (2005).
1. 池口満徳、桑島邦博、"フォールディング実験とその分子動力学による再現"、現代化学、407, 49-54 (2005)
1. 渕上壮太郎、 第6回蛋白質立体構造予測コンテストCASP6に参加して、 蛋白質核酸酵素 Vol.50 No.5 471-476 (2005)
1. 木寺詔紀、 二重動的計画法、分子動力学・モンテカルロ法、弾性ネットワーク、 バイオインフォマティクス事典、 共立出版 （2005）.
1. 木寺詔紀、 蛋白質間相互作用、 生物物理学ハンドブック、 朝倉書店 II-10 (2005).
1. 木寺詔紀、池口満徳、 構造ダイナミクスの理論とシミュレーション、 タンパク質科学－構造・物性・機能 5.2、 化学同人 (2005).

# 2004
1. Y. Hioki, K. Ogasahara, S.J. Lee, J. Ma, M. Ishida, Y. Yamagata, Y. Matsuura, M. Ota, M. Ikeguchi, S. Kuramitsu and K. Yutani, "The crystal structure of the tryptophan synthase b2 subunit from the hyperthermophile Pyrococcus furiosus Investigation of stabilization factors", Eur. J. Biochem., 271, 2624-2635 (2004).
1. M. Ikeguchi, "Partial Rigid-Body Dynamics in NPT, NPAT and NPγT Ensembles for Proteins and Membranes", J. Comput. Chem., 25, 529-541 (2004).
1. J. G. Kim, Y. Fukunishi, A. Kidera and H. Nakamura, "Stochastic formulation of sampling dynamics in generalized ensemble methods.", Physical Review E 69, 021101 1-10 (2004).
1. K. Moritsugu and A. Kidera, "Protein motions represented in moving normal mode coordinates.", J. Phys. Chem. B. 108, 3890-3898 (2004).
1. R. Koike, K. Kinoshita and A. Kidera, "Probabilistic description of protein alignments for sequences and structures.", Proteins: Structure, Function, and Bioinformatics 56, 157-166 (2004).
1. J. G. Kim, Y. Fukunishi, A. Kidera and H. Nakamura, "Generalized simulated tempering realized on expanded ensembles of non-Boltzmann weights.", J. Chem. Phys. 121, 5590-5601 (2004).
1. M. Ota, M. Ikeguchi and A. Kidera, "Phylogeny of protein folding trajectories reveals a unique pathway to native structure.", Proc. Natl. Acad. Sci. USA 101, 17658-17663 (2004).
1. 木寺詔紀、 ナノバイオロジー（書評）、 蛋白質核酸酵素 49, 2269-2270 (2004).


# 2003
1. Y. Akamatsu, D. Dziadkowiec, M. Ikeguchi, H. Shinagawa and H. Iwasaki, "Two different Swi5-containing protein complexes are involved in mating-type switching and recombination repair in fission yeast", Proc. Natl. Acad. Sci. USA, 100, 15770-15775 (2003).
1. S. Nakamura, M. Ikeguchi, K. Shimizu, "Dynamical Analysis of tRNAGln-GlnRS Complex using Normal Mode Calculation", Chem. Phys. Lett., 372, 423-431 (2003).
1.　S. Fuchigami and K. Someda, "Nonadiabatic Dynamics of the Classical Hydrogen Molecular Ion H2+", J. Phys. Soc. Jpn. 72, 1891-1898 (2003).
1.　T. Terada, Y. Matsuo and A. Kidera, "A method for evaluating multicanonical potential function without iterative refinement: Application to conformational sampling of a globular protein in water.", J. Chem. Phys 118, 4306-4311 (2003) Erratum: J. Chem. Phys 118, 9899 (2003).
1.　R. Koike, K. Kinoshita and A. Kidera, "Ring and zipper formation is the key to understanding the structural variety in all beta proteins.", FEBS Lett. 533, 9-13 (2003).
1.　K. Moritsugu, O. Miyashita and A. Kidera, "Temperature dependence of vibrational energy transfer in a protein molecule.", J. Phys. Chem. B 107, 3309-3317 (2003).
1.　J. G. Kim, Y. Fukunishi, A. Kidera and H. Nakamura, "Determination of multicanonical weight based on a stochastic model of sampling dynamics.", Physical Review E 68, 021110 1-8 (2003).
1.　木寺詔紀、池口満徳、"蛋白質機能発現のダイナミクス的側面"、日本物理学会誌、58, 246-253 (2003).

# 2002
1. S. Nakamura, D. Kyono, M. Ikeguchi, and K. Shimizu, "A New Method for Parallel Computation of Hessian Matrix of Conformational Energy Function in Internal Coordinates", J. Comput. Chem., 23, 463-469 (2002).
1. J-G. Jee, T. Ikegami, M. Hashimoto, T. Kawabata, M. Ikeguchi, Takeshi Watanabe and M. Shirakawa, "Solution Structure of the Fibronectin Type III Domain from Bacillus circulans WL-12 Chitinase A1", J. Biol. Chem., 277, 1388-1397 (2002).
1. T. Terada and A. Kidera, "A generalized form of the conserved quantity in constant-temperature molecular dynamics.", J. Chem. Phys. 116, 33-41 (2002).
1. Y. Joti, M. Nakasako, A. Kidera and N. Go, "Nonlinear temperature dependence of crystal structure of Lysozyme: Correlation between coordinate shifts and thermal factors.", Acta Cryst. Sect. D 58, 1421-1432 (2002).
1. 木寺詔紀、池口満徳、 蛋白質構造の多様性とダイナミクス、 蛋白質核酸酵素 47, 1052-1057 (2002).
1. 池口満徳、"タンパク質変性と変性剤の役割－疎水効果に対する尿素の影響－", 生物物理、42, 72-74 (2002).
1.　木寺詔紀、 立体構造の類似性を知る、 あなたにも役立つ　バイオインフォマティクス　第8章、 共立出版　61-68 (2002).
1.　木寺詔紀、 タンパク質結晶構造解析に見るダイナミクス、 シリーズ・ニューバイオ フィジックス II「生物物理学とはなにか　－未解決問題への挑戦－」第1章1-2タンパク質、 共立出版 (2002).
1.　池口満徳、木寺詔紀、 化学フロンティア⑦ 生体系のコンピュータシミュレーション、 17章 イオンポンプ、能動輸送、岡崎進、岡本祐幸編、 化学同人 （2002）.

# 2001
1. S.-H. Chong, Y. Joti, A. Kidera, N. Go, A. Ostermann, A. Gassmann and F. Parak, "Dynamical transition of Myoglobin in a crystal. Comparative studies of X-ray crystallography and Mossbauer spectroscopy.", European Biophysics J. 30, 319-329 (2001).
1. H. Yamashita and A. Kidera, "Environmental influence on electron scattering from a molecule.", Acta Cryst. Sect A 57, 518-525 (2001).
1. H. Yamashita, S. Endo, H. Wako and A. Kidera, "Sampling efficiency of molecular dynamics and Monte Carlo method in protein simulation.", Chem. Phys. Lett. 342, 382-386 (2001).
1. K. Kinoshita, J. Furui and H. Nakamura, "Identification of Protein Functions from a Molecular Surface Database, eF-site,", JSFG, 2, 1-14 (2001).
1. A.Tanaka, Y. Ikeda, S. Abe-Dohmae, R. Arakawa, K. Sadanami, A. Kidera, S. Nakagawa, T. Nagase, R. Aoki, N. Kioka, T. Amachi, S. Yokoyama, and K. Ueda, "Human ABCA1 Contains a Large Amino-Terminal Extracellular Domain Homologous to an Epitope of Sjogren's Syndrome", Biochem Biophys Res Commun., 283,1019-25 (2001).
1. M. Ikeguchi, S. Nakamura, and K. Shimizu, "Molecular Dynamics Study on Hydrophobic Effects in Aqueous Urea Solutions", J. Am. Chem. Soc., 123, 667-682 (2001).
1. 木寺詔紀、 立体構造の類似性を知る、 蛋白質核酸酵素 46, 2198-2204 (2001).
1. 木寺詔紀、 タンパク質のダイナミクス－バクテリオロドプシンを中心にして、 化学 56, 70-71 (2001).

# 2000
1. N. Nakajima, J. Higo, A. Kidera, and H. Nakamura, "Free energy landscapes of short peptides by enhanced conformational sampling," J. Mol. Biol., 296, 197-218 (2000).
1. A. R. Kinjo, A. Kidera, H. Nakamura, and K. Nishikawa, "Physico-chemical evaluation of protein folds predicted by threading," European Biophysics J., 30, 1-10 (2000)
1. Y. Ikeda, M. Inaki, A. Kidera, and H. Hayashi, "Physically crosslinked elastomers prepared from oligomeric polyolefins with mesogenic units.," J. Polymer Science, Part B: Plymer Physics, 38, 2247-2253 (2000).
1. K. Moritsugu, O. Miyashita and A. Kidera, "Vibrational Energy Transfer in a Protein Molecule," Phys. Rev. Lett., 85, 3970-3973 (2000).
1. S. Takasaki, M. Sekijima, S. Nakamura, M. Ikeguchi and K. Shimizu, "Scheduling Policy and Mechanism of Parsley, A Parallel Programming Environment," Proceedings of 12th IASTED International Conference PDCS, 605-610, (2000).
1. M. Yoshikawa, H. Iwasaki, K. Kinoshita and H. Shinagawa, "Two basic residues, Lys-107, and Lys-118, of RuvC resolvase are involved in critical contacts with the Holliday junction for its resolution," Genes to Cells, 5 803-813, (2000)

# 1999
1. K. Kinoshita, A. Kidera, and N. Go, "Structural motif of phosphate binding site common to various protein superfamilies: All-against-all structural comparison of protein-mononucleotide complexes," Protein Engineering. 12, 11-14 (1999).
1. K. Mitsuoka, T. Hirai, K. Murata, A. Miyazawa, A. Kidera, Y. Kimura, and Y. Fujiyoshi, "Charge distribution in the refined structure of bacteriorhodopsin based on electron crystallography,"J. Mol. Biol., 286, 861-882 (1999)
1. K. Kinoshita, A. Kidera, and N. Go, "Diversity of functions of proteins with internal symmetry in spatial arrangement of secondary structural elements," Protein Science, 8, 1210-1217 (1999).
1. H. Shirai, A. Kidera, H. Nakamura, "H3-Rule: Identification of CDR-H3 structures in antibodies,"FEBS Lett. 455, 188-197 (1999).
1. A. Kidera, "Smart Monte Carlo simulation of a globular protein,"Int. J. Quantum Chem. 75, 207-214 (1999)
1. K Kinoshita, A. Kidera, and N. Go, "Symmetry in protein folds: Implication in evolution and folding," In "Old and New Views of Protein Folding", Proceeding of Taniguchi International Symposium, Elsevire Science, 249-257, (1999).
1. F. Parak, A. Ostermann, A. Gassmann, C. Scherk, S.-H. Chong, A. Kidera, and N. Go, "Biomolecules: Fluctuations and relaxations," in "BIOLOGICAL PHYSICS: Third International Symposium", AIP Conference Proceedings, 487, 117-127(1999)
1. T. Ohkawa, D. Namihira, T. Maeda, N. Komoda, A. Kidera, and H. Nakamura, "Classification and Feature Extraction of Protein Structures Based on Structure Transformation," International J. Artificial Intelligence Tools, 8, 401-418 (1999).
1. S. Shimizu, M. Ikeguchi, S. Nakamura and K. Shimizu, "Size Dependence of Transfer Free Energies:a Hard-Sphere-Chain-Based Formalism,"J. Chem. Phys., 110, 2971-2982 (1999).
1. S. Nakamura, M. Ikeguchi and K. Shimizu, "Structural Modeling of DNA Mini-Hairpin Molecules with Various Loop Sequences," Chem. Phys. Lett., 308, 267-273 (1999).
1. M. Ikeguchi, S. Nakamura, and K. Shimizu, "Hydrophobic Effects: Roles of Water and Denaturants," in "Old and New Views of Protein Folding", Proceedings of Taniguchi International Symposium, Elsevier Science B.V., 195-204 (1999).
1. M. Sekijima, S. Takasaki, S. Nakamura, M. Ikeguchi and K. Shimizu, "Parsley: A Scalable Framework for Dependence-Driven Task Scheduling in Distributed-Memory Multiprocessor Systems," Proceedings of the IASTED International Conference PDCS, 800-805, (1999).

# 1998
1. H. Ishida, Y. Nagai, and A. Kidera, "Symplectic Integrators for Molecular Dynamics of Protein in Water," Chem. Phys. Lett. 282, 115-120 (1998).
1. H. Shirai, N. Nakajima, J. Higo, A. Kidera, and H. Nakamura, "Conformational sampling of CDR-H3 in antibodies by multicanonical molecular dynamics simulation,"J. Mol. Biol. 278, 481-496 (1998)
1. K. Maenaka, M. Matsushima, G. Kawai, A. Kidera, K. Watanabe, R. Kuroki, and I. Kumagai, "Structural and functional effect of Trp-62 Gly and Asp-101 Gly substitutions on substrate-binding modes of mutant hen egg-white lysozymes," Biochemical J. 333, 71-76 (1998)
1. H. Ishida, Y. Jochi, and A. Kidera, "Dynamic structure of subtilisin-eglin C complex studied by normal mode," Proteins: Struct. Funct. Genet. 32, 324-333 (1998).
1. H. Ishida and A.Kidera, "Canonical molecular dynamics of a protein in water by high-order decomposition of Liouville operator," J. Chem. Phys. 109, 3276-3284 (1998).
1. T. Maeda, K. Kamada, T.Ohkawa, N.Komoda, H. Nakamura, and A.Kidera, "Feature extraction of protein folds based on secondary structure transformation", Proc. IEEE International Joint Symposia on Intelligence and Systems, 158-162 (1998) May 21-23, 1998. Rockville, Maryland, IEEE Comput. Soc. Press
1. M. Ikeguchi, S. Shimizu, S. Nakamura and K. Shimizu, "Roles of Hydrogen Bonding and the Hard Core of Water on Hydrophobic Hydration," J. Phys. Chem., 102, 5891-5898 (1998).
1. S. Nakamura, M. Ikeguchi and K. Shimizu, "Parallel Algorithm for Efficient Calculations of Second Derivatives of Conformational Energy Function in Internal Coordinates," J. Comp. Chem., 18, 1716-1723 (1998).
1. M. Ikeguchi, S. Shimizu, K. Tazaki, S. Nakamura and K. Shimizu, "Calculation of Temperature Dependence of Free Energy caused by Potential Function Changes," Chem. Phys. Lett., 288, 333-337 (1998).
1. S. Shimizu, M. Ikeguchi and K. Shimizu, "An Off-lattice Theory of Solvation: Extension of the Flory χParameter into Continuum Space,"Chem. Phys. Lett., 282, 79-90, (1998).
1. S. Shimizu, M. Ikeguchi, S. Nakamura and K. Shimizu, "Molecular Volume, Surface Area, and Curvature Dependence of the Configurational Entropy Change upon Solvation: Effects of Molecular Bonding," Chem. Phys. Lett., 284, 235-246, (1998).
1. J. Sagara, S. Shimizu, T. Kawabata, S. Nakamura, M. Ikeguchi and K. Shimizu, "The Use of Sequence Comparison to Detect Identities in tRNA Genes," Nucleic Acid Research, 26, 1974-1979 (1998).
1. T. Murata, M. Sekijima, H. Masaki, S. Nakamura, M. Ikeguchi and K. Shimizu, "Parallel Programming Environment with Dependence-Driven Subtask Scheduling: Design and Application to Molecular Dynamics Simulation," Proceedings of the 2nd IASTED International Conference PDCN, 650-653, (1998).

# 1997
1. A. Kidera, T. Higashira, Y. Ikeda, K. Urayama, and S. Kohjiya, "GPC analysis of polymer network formation: 2. Bifunctional PPG prepolymer/ crosslinker system," Polymer Bulletin 38, 461-468 (1997)
1. Y. Kimura, D. G. Vassylyev, A. Miyazawa, A. Kidera, M. Matsushima, K. Mitsuoka, K. Murata, T. Hirai, and Y. Fujiyoshi, "Membrane surface structure and proton pathway of bacteriorhodopsin analyzed by electron crystallography", Nature 389, 206-211 (1997).
1. Y. Kimura, D. G. Vassylyev, A. Miyazawa, A. Kidera, M. Matsushima, K. Mitsuoka, K. Murata, T. Hirai, and Y. Fujiyoshi, "High resolution structure of bacteriorhodopsin determined by electron crystallography," Photochemistry and Photobiology, 66, 407-412 (1997)
1. K. Urayama, S. Kohjiya, M. Yamamoto, Y. Ikeda, and A. Kidera, "Formation process of end-linked networks by gel permeation chromatogrtaphy," J. Chem. Soc. Faraday Trans. 93, 3689-3693 (1997).
1. J. Higo, N. Nakajima, H. Shirai, A. Kidera, and H. Nakamura, "Two-component, multicanonical Monte Carlo method for effective conformation sampling", J. Comput. Chem. 18, 2086-2092 (1997).
1. N. Nakajima, J. Higo, A. Kidera, and H. Nakamura, "Flexible docking of a ligand peptide to a receptor protein by multicanocnical molecular dynamics simulation", Chem. Phys. Lett., 278, 297-301 (1997) .
1. S. Shimizu, M. Ikeguchi and K. Shimizu, "Extracting Contact Free Energy from Solubility: Excluded Volume Effects of Polymers in Continuum Space," Chem. Phys. Lett., 268, 93-100 (1997).
1. K. Shimizu, A. Oishi, H. Ashihara, M. Ikeguchi and S. Nakamura, "Design and Implementation of Parallel Programming Environment with Dynamic Resource Management Services," Proceedings of the 12th ISCA International Conference on Computers and Their Applications, 226-231, (1997).

# 1996
1. S. Kohjiya, Y. Takada, K. Urayama,, Y. Tezuka, and A. Kidera, "GPC analysis of polymer network formation: 1. Bifunctional siloxane monomer/crosslinker system", Bull.Chem.Soc.Jpn. 69, 565-574 (1996)
1. T. Yamada and A. Kidera, "Tailoring echistatin to possess higher affinity for integrin ?IIb?3.", FEBS Lett. 387, 11-15 (1996).
1. H. Nakamura, R. Tanimura, and A. Kidera, "Side-chain conformations cooperatively restricted in protein secondary structure. I. A novel method for exhaustive structure searching", Proc Jpn Acad [B] 72, 143-148 (1996).
1. H. Nakamura, R. Tanimura, and A. Kidera, "Side-chain conformations cooperatively restricted in protein secondary structure. II. Side-chain configurational entropies of alpha -helices in the folding nuclei",Proc Jpn Acad [B] 72, 149-152 (1996).
1. K. Nagai and A. Kidera, "A comparative study of numerical integrator on one-dimensional harmonic oscillator", KEIEI KEIRI 19, 23-43 (1996)
1. S. Iwai, M. Wakasa, E. Ohtsuka, S. Kanaya, A. Kidera and H. Nakamura, "Interaction of the basic protrusion of Escherichia coli ribonuclease HI with its substrate", J. Mol. Biol. 263, 699-706 (1996)
1. N. Nakajima, H. Nakamura, and A. Kidera, "Multicanonical ensemble generated by molecular dynamics simulation for enhanced conformational sampling of peptides", J. Phys. Chem. 101, 817-824 (1996)
1. T. Ookawa, D. Namihira, N. Komoda, A. Kidera, H. Nakamura, "Protein structure classification by structure transformation", Proc.IEEE International Joint Symposia on Intelligence and Systems, 23-29 (1996) November 4-5, 1996. Rockville, Maryland, IEEE Comput. Soc. Press
1. H. Shirai, A. Kidera, and H. Nakamura, "Structural classification of CDR-H3 in antibodies", FEBS Lett. 399,1-8 (1996).

# 1995
1. A. Kidera, "Enhanced conformational sampling in Monte Carlo simulations of proteins: Application to a constrained peptide", Proc. Nat. Acad. Sci. U.S.A. 92, 9886-9889 (1995).
1. S. Nakamura, H. Hirose, M. Ikeguchi and J. Doi, "Conformational Energy Minimization using a Two-Stage Method," J. Phys. Chem., 99, 8374-8378, (1995).
1. M. Ikeguchi and J. Doi, "Direct Numerical Solution of the Ornstein-Zernike Integral Equation and Spatial Distribution of Water around Hydrophobic Molecules," J Chem. Phys., 102, 5011-5017, (1995).
1. T. Izumitani, K. Tazaki, T. Kawabata, M. Ikeguchi, J. Doi, "Conformational Path of TRP Repressor, in "Perspective on Protein Engineering & Complementary Technologies", Mayflower Worldwide Limited, 45-48, (1995)
1. H. Hirose, S. Nakamura, M. Ikeguchi, and J. Doi, "Improved Diffusion Equation Method for Large Molecules," in "Perspective on Protein Engineering & Complementary Technologies", Mayflower Worldwide Limited, 49-52, (1995)
